packages/components/src/Chip/Chip.tsx

```diff
@@ -44,17 +48,24 @@ export const Chip = ({
 
   return (
     <Tooltip message={tooltipMessage}>
-      <button
+      <Tag
         className={classes}
-        onClick={(ev: React.MouseEvent<HTMLButtonElement>) =>
-          onClick && onClick(value, ev)
+        onClick={(ev: React.MouseEvent<HTMLButtonElement | HTMLDivElement>) =>
+          onClick &&
+          !actAsFormElement &&
+          onClick(value || label, ev as React.MouseEvent<HTMLButtonElement>)
         }
         tabIndex={tabIndex}
-        onKeyDown={onKeyDown}
+        onKeyDown={(
+          ev: React.KeyboardEvent<HTMLButtonElement | HTMLDivElement>,
+        ) =>
+          onKeyDown && onKeyDown(ev as React.KeyboardEvent<HTMLButtonElement>)
+        }
         aria-label={ariaLabel || label}
         disabled={disabled}
         role={role}
         type="button"
+        data-testid={dataTestID || "chip-wrapper"}
```

Comment:
Something else I learned about today is the existence of [this wiki](https://jobber.atlassian.net/wiki/spaces/TEAMATL/pages/2476933462/How+should+we+declare+test+ID+s+in+Atlantis+components)

There are a ton of older components that are writing the testID the way you did but in `Combobox` and `DataList` we are using the convention described in that wiki.  So I guess this should instead be `ATL-Chip`?

Reply:
In this particular case it's because I noticed references to `chip-wrapper` in the existing tests, and I thought someone else out there might have done the same, so in this case it's a compatibility thing which is why I kept it as chip-wrapper instead of changing it to the new pattern. If this were a whole new component I would agree to update the ID to be the new pattern.

If we're convinced though that updating this id is non-breaking, I'd be happy to update it both here and in the tests to be inline with Combobox/Datalist.

Reply:
Ahh ok good point - nah leave it as is! Maybe one day we can do a cleanup because there's more components **not** using the `ATL` format than there is right now.